tests/imposter/private-api.yaml

```diff
@@ -124,8 +124,6 @@ paths:
           description: OK
         "400":
           description: Bad Request
-        "422":
```

Comment:
Remember, this file is a copy-and-paste of https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/main/infrastructure/private-api.yaml

Reply:
Good catch